molecule_azure/cookiecutter/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/destroy.yml

```diff
@@ -10,50 +10,12 @@
     virtual_network_name: molecule_vnet
     subnet_name: molecule_subnet
   tasks:
-    - name: Destroy molecule instance(s)
-      azure_rm_virtualmachine:
-        resource_group: "{{ resource_group_name }}"
-        name: "{{ item.name }}"
-        state: absent
-        remove_on_absent:
-          - all_autocreated
-      register: server
-      with_items: "{{ molecule_yml.platforms }}"
-      async: 7200
-      poll: 0
-
-    - name: Wait for instance(s) deletion to complete
-      async_status:
-        jid: "{{ item.ansible_job_id }}"
-      register: azure_jobs
-      until: azure_jobs.finished
-      retries: 300
-      with_items: "{{ server.results }}"
-      # avoid failure when resource_group is already missing, as we remove it later anyway
-      failed_when: false
-
-    - name: Destroy subnet
-      azure_rm_subnet:
-        resource_group: "{{ resource_group_name }}"
-        name: "{{ subnet_name }}"
-        virtual_network_name: "{{ virtual_network_name }}"
-        state: absent
-      # avoid failure when resource_group is already missing, as we remove it later anyway
-      failed_when: false
-
-    - name: Destroy virtual network
-      azure_rm_virtualnetwork:
-        resource_group: "{{ resource_group_name }}"
-        name: "{{ virtual_network_name }}"
-        state: absent
-      # avoid failure when resource_group is already missing, as we remove it later anyway
-      failed_when: false
-
-    - name: Destroy resource group
+    - name: Destroy resource group and all associated resources
       azure_rm_resourcegroup:
         name: "{{ resource_group_name }}"
         state: absent
-        force_delete_nonempty: true
+        force_delete_nonempty: yes
```

Comment:
I am sure yamllint would complain about not using recommended yaml booleans, please use them, even if ansible team does not want to conform to newer YAML standard.

Reply:
I checked yamllint on destroy.yml , its not failing , but i think we should use `true`. I will make that changes